planning/costmap_generator/test/test_points_to_costmap.cpp

```diff
@@ -0,0 +1,225 @@
+// Copyright 2022 The Autoware Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <costmap_generator/points_to_costmap.hpp>
+
+#include <gtest/gtest.h>
+using pointcloud = pcl::PointCloud<pcl::PointXYZ>;
+class PointsToCostmapTest : public ::testing::Test
+{
+protected:
+  void SetUp() override { rclcpp::init(0, nullptr); }
+
+  ~PointsToCostmapTest() override { rclcpp::shutdown(); }
+
+  grid_map::GridMap construct_gridmap();
+
+public:
+  double grid_resolution_ = 1;
+  double grid_length_x_ = 20;
+  double grid_length_y_ = 20;
+  double grid_position_x_ = 0;
+  double grid_position_y_ = 0;
```

Comment:
- [nits] Omitting the fractional part is accepted but not recommended.
- `constexpr` (or `const`) is always recommended.

```suggestion
  constexpr double grid_resolution_ = 1.0;
  constexpr double grid_length_x_ = 20.0;
  constexpr double grid_length_y_ = 20.0;
  constexpr double grid_position_x_ = 0.0;
  constexpr double grid_position_y_ = 0.0;
```

References:
https://google.github.io/styleguide/cppguide.html#Floating_Literals
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es25-declare-an-object-const-or-constexpr-unless-you-want-to-modify-its-value-later-on